src/util/log.c

```diff
@@ -6,8 +6,14 @@
 #define LOGMODULE log
 #include "log.h"
 
+#if !defined(_MSC_VER) || defined(__INTEL_COMPILER)
```

Comment:
Will this also cover Visual studio using clang?

Reply:
Yes.  We only use clang. @flihp said that's because the MS Visual Studio C compiler does not support C99, so we have to use clang.